modules/router/manifests/nginx.pp

```diff
@@ -125,6 +125,13 @@
     require => File['/usr/share/nginx/www'],
   }
 
+  # HTML verification for DWP YouTube channel
+  file { '/usr/share/nginx/www/google6db9c061ce178960.html':
+    ensure  => present,
+    source  => 'puppet:///modules/router/google6db9c061ce178960.html',
```

Comment:
I was going to do that but having read the documentation from Google, I suspect that the file may have contents we need to add. If not, I'll come back and tidy this up after.